options/options.go

```diff
@@ -306,6 +306,12 @@ type TerragruntOptions struct {
 	// Files with variables to be used in modules scaffolding.
 	ScaffoldVarFiles []string
 
+	// Do not include root unit in scaffolding.
+	ScaffoldNoIncludeRoot bool
+
+	// Name of the root Terragrunt configuration file, if used.
+	ScaffoldRootFileName string
```

Comment:
I think `Clone()` should be updated

Reply:
Why would these need to be cloned? Scaffold isn't used in `run-all`.

Reply:
I added it just to keep this moving.